frontend/src/components/school/SchoolDetails.vue

```diff
@@ -0,0 +1,501 @@
+        break;
+      default:
+        schoolOrg = '-';
+      }
+
+      return schoolOrg;
+    },
+    getNLCActivity(neighborhoodLearning){
+      let NLCActivity = '';
+      for(const nl of neighborhoodLearning){
+        switch(nl){
+        case 'EARLYLEARN':
+          NLCActivity = 'Early Learning';
+          break;
+        case 'AFTERSCHL':
+          NLCActivity = 'After School Programs';
+          break;
+        case 'CONTINEDUC':
+          NLCActivity = 'Continuing Education';
+          break;
+        case 'SENIORS':
+          NLCActivity = 'Seniors';
+          break;
+        case 'SPORTRECR':
+          NLCActivity = 'Sports and Recreation';
+          break;
+        case 'COMMUNITY':
+          NLCActivity = 'Community Use';
+          break;
+        case 'INTEGRSERV':
+          NLCActivity = 'Integrated Services';
+          break;
+        default:
+          NLCActivity = '-';
+        }
+      }
+
+      return NLCActivity;
+    },
+    getFacilityType(school){
+      let type = null;
+      if(school.facilityTypeCode === 'STANDARD'){
+        type = 'Standard School';
+      } else if(school.facilityTypeCode === 'OFFSHORE'){
+        type = 'Offshore School';
+      } else if(school.facilityTypeCode === 'DIST_LEARN'){
+        type = 'Distance Learning';
+      }
+      return type;
+    },
+    getSchoolCategory(school){
+      let category = null;
+      if(school.schoolCategoryCode === 'PUBLIC'){
+        category = 'Public School';
+      } else if(school.schoolCategoryCode === 'INDEPEND'){
+        category = 'Independent School';
+      } else if(school.schoolCategoryCode === 'YUKON'){
+        category = 'Yukon School';
+      } else if(school.schoolCategoryCode === 'FED_BAND'){
+        category = 'Indigenous School';
+      } else if(school.schoolCategoryCode === 'OFFSHORE'){
+        category = 'Offshore School';
+      }
+      return category;
+    },
+    getSchoolStatus(school) {
+      const currentDate = new Date();
+      let openedDate = school.openedDate;
+      let closedDate = school.closedDate;
+      let status = null;
+
+      if (openedDate <= currentDate || closedDate === null || closedDate > currentDate) {
+        status = 'Open';
+      } else if (openedDate > currentDate) {
+        status = 'Opening';
+      } else if (closedDate > currentDate) {
+        status = 'Closing';
+      } else {
+        status = 'Closed';
+      }
+
+      return status;
+    },
+    getStatusColor(status) {
+      if (status === 'Open') {
+        return 'green';
+      } else if (status === 'Opening'){
+        return 'blue';
+      } else if (status === 'Closing'){
+        return 'orange';
+      } else if (status === 'Closed') {
+        return 'red';
+      }
+    },
+    formatDate(date){
+      if(date !== '' && date !== null) {
+        return new Date(date).toISOString().slice(0, 10).replace(/-/g, '/');
+      } else {
+        return '-';
+      }
+    },
+    formatPhoneNumber(phNumber){
```

Comment:
There's a formatPhoneNumber function in the shared utils. https://github.com/bcgov/EDUC-STUDENT-ADMIN/blob/master/frontend/src/utils/format.js#:~:text=%7D-,export%20function%20formatPhoneNumber(phoneNumber)%20%7B,-if%20(!

Would suggest that we can put the formatDate function in this file too

Reply:
That formatDate function returns a date that looks like the original including the time that we don't want.